app/javascript/components/LinkButton.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <button
+    @click="moveMe()"
+    :class="[classmod,'text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
+    :disabled="fieldDisabled"
+  >{{ fieldPlaceholder }}</button>
```

Comment:
Lo otro, las classes siempre debiesen ir definidas aquí, no usando classmod (o solo para cosas pequeñas)
```suggestion
  <button
    @click="moveMe()"
    :class="[classmod,'bg-blue-500 hover:bg-blue-700 mx-4 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
    :disabled="fieldDisabled"
  >{{ fieldPlaceholder }}</button>
```